arch/arm/src/armv7-r/arm_syscall.c

```diff
@@ -434,11 +434,31 @@ uint32_t *arm_syscall(uint32_t *regs)
 
           if (rtcb->xcp.kstack != NULL)
             {
-              DEBUGASSERT(rtcb->xcp.kstkptr == NULL &&
-                          rtcb->xcp.ustkptr != NULL);
+              uint32_t *usp;
```

Comment:
need change to uint8_t *?

Reply:
Yes you are correct, this will create a frame that is 4/8 times too big